src/client/filter.py

```diff
@@ -224,13 +226,21 @@ class Key(ExtendableStrEnum):
 
 
 class Skill(StatefulComparableConditionMixin, Condition, order=False, eq=False, kw_only=False, spec_value=Condition.Category.SKILL):
-    """Use to select Tolokers by skill value.
+    """Filtering Tolokers by skills.
+
+    Pass the ID of a skill to the filter constructor.
+    To select Tolokers without a skill, compare created filter with `None`.
+
+    This filter belongs to the `skill` category.
```

Comment:
Didn't understand this. What is a category? Also there is no mentions of categories for other filters.

Reply:
I mean the `category` parameter in the API.
https://toloka.ai/en/docs/api/concepts/filters
Also mentioned `category` in `Profile` and `Computed` classes.

It was not clear for you, so it's better to remove `category` from the `Skill`.

In `Profile` and `Computed` `category` is not confusing, is it?

Reply:
> It was not clear for you, so it's better to remove `category` from the `Skill`.
I guess so, as a user does not need to know about `category` API field when using toloka-kit.

> In `Profile` and `Computed` category is not confusing, is it?
Yes, because of the context